ui-lib/components/Layout/Modal/index.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React from "react";
+import styled from "styled-components";
+import type { ThemedComponent } from "@ui/styles/StyleProvider";
+import Button from "@ui/components/Button";
+import Cross from "@ui/icons/Cross";
+
+type Props = {
+  children: React$Node,
+  isOpen: boolean,
+  onClose?: () => void,
+  width?: number,
+  height?: number,
+};
+
+const Container: ThemedComponent<*> = styled.div`
+  position: relative;
+  width: 100%;
+  height: 100%;
+  background-color: ${p => p.theme.colors.palette.v2.background.default};
+  padding: ${p => p.theme.space[3]}px;
+`;
+
+const Wrapper: ThemedComponent<{ width?: number, height?: number }> = styled.div`
+  height: ${p => p.height || p.theme.sizes.modal.min.height}px;
+  width: ${p => p.width || p.theme.sizes.modal.min.width}px;
+  min-height: ${p => p.theme.sizes.modal.min.height}px;
+  min-width: ${p => p.theme.sizes.modal.min.width}px;
+  max-height: ${p => p.theme.sizes.modal.max.height}px;
+  max-width: ${p => p.theme.sizes.modal.max.width}px;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  justify-content: space-between;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform translate(-50%, -50%);
+  z-index: 1000;
```

Comment:
Last week during the dedicated meeting design we asked the design team to exposes us standardized values like padding, margin etc... Maybe that is already planned but if not, that would be useful to ask them to standardise z-index values.